src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -2127,6 +2128,10 @@ in src-script.js and main.js
 		padding: 2px 4px;
 		box-shadow: 0 0 4px var(--main-background-color);
 	}
+
+	.item-table > li > .item-name {
+		max-width: 40vw;
```

Comment:
I just realized: why using the viewport width and not the parent element width?

Reply:
The parents width element is not static so I can't make the css `50%`

Reply:
The viewport width isn't either? Not sure to understand what you mean.

Reply:
When the item-name gets a width of 50% nothing changes. I'm assuming this is due to the fact that the parent doesn't have a predefined width. Instead I've set it to 40vw which seems to scale pretty nicely.

Reply:
That doesn't make any sense. The parent has a width. Also: you can resize the sidebar, changing the parent element width, but not the viewport width.

Reply:
Upon interacting with the parents width, using something like `max-width: 50%` on `.item-name`, no change occurs. I'm not sure exactly why this is the case but I'm guessing it has to do with how the parents width is defined.